PROJECT_CHARTER.md

```diff
@@ -67,7 +67,35 @@ Section Intentionally Left Blank
 
 ### Section 4.2: Decision-making, Voting, and/or Elections
 
-Section Intentionally Left Blank
+**Decision-making**
+
+The Fastify's features can be discussed in the GitHub's issues and/or projects.
+The consensus on a discussion is reached when:
+
+- the majority of the Collaborator agree on the decision
+- there is not any blocking concern by the Lead Maintainers
+
+Whenever there is not consensus, Lead Maintainers will have final say on the topic.
+
+**Voting, and/or Elections**
+
+Individuals making significant and valuable contributions to the project may be candidate to join
+the Fastify's org.
+
+A Fastify Team Member needs to open a private team discussion on GitHub and list the candidates who
+wants to sponsor with a link to the user's contributions (eg: `[USERNAME](https://github.com/search?q=author:USERNAME+org:fastify)`).
+
+Otherwise, a Contributor may self-apply if he/she thinks to meet the above criteria reaching out
+to a Lead Maintainer privately with a few links to his/her valuable contributions.
+The Lead Maintainer will reply to the Contributor and will decide if candidate he/she to the FTM.
+
+The consensus to a new candidate to join the FTM is reached when:
+
+- at least one Lead Maintainers approve
+- at least two Team Members approve
+- no one contrary
```

Comment:
```suggestion
- at least one of the Lead Maintainers approve
- at least two of the Team Members approve
- no one objects
```

Is it possible for a single objection to overrule the first two items?

Reply:
> Is it possible for a single objection to overrule the first two items?

Theoretically yes.
Do you agree?

Reply:
No. As long as the first two items are met then the third should not matter. If I dissent on an item, but everyone else is on board with it, then I am outvoted and should accept that.

@fastify/core